test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -8,9 +8,13 @@ function testMultiplyFractionsSearch(exprString, outputStr) {
 
 describe('multiplyFractions', function () {
```

Comment:
but then there's cancelling code and also... other cancelling code? do you think we could just merge them or something?